googleapiclient/discovery_cache/__init__.py

```diff
@@ -19,6 +19,9 @@
 import logging
 import datetime
 
+
+logger = logging.getLogger(__name__)
```

Comment:
Since this is a module-scope constant its name should be styled as `LOGGER`, right? (Here and throughout this change.)